<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUniversalPromptDlg</class>
 <widget class="QDialog" name="CUniversalPromptDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>提示信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CCustomHeadBar" name="chb_head_bar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lb_title">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>14</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size:14px;
background-color: transparent;
font-weight:400;
color:rgba(74,177,52,1);</string>
        </property>
        <property name="text">
         <string>提示</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pb_close">
        <property name="minimumSize">
         <size>
          <width>11</width>
          <height>12</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>11</width>
          <height>12</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
padding: 0px;
border: noborder;
border-radius: 0px;
background-image: url(:/TQLive/images/close_normal.png);
}
QPushButton:hover {
background-image: url(:/TQLive/images/close_pressed.png);
}
QPushButton:pressed {
background-image: url(:/TQLive/images/close_pressed.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="lb_prompt_msg">
       <property name="styleSheet">
        <string notr="true">font-size:14px;
background-color: rgba(38, 38, 38, 1);
font-weight:400;
color:rgba(220,220,220,1);</string>
       </property>
       <property name="text">
        <string>提示信息</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_ok">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 rgba(74,177,52,1), stop:1 rgba(115,194,14,1));
border-radius:12px;
font-size:12px;
font-weight:400;
color:rgba(255,255,255,1);
padding:0px;</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CCustomHeadBar</class>
   <extends>QWidget</extends>
   <header>ccustomheadbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
